aio/content/guide/upgrade-lazy.md

```diff
@@ -272,28 +270,28 @@ angular.module('mainAngularJsModule', [
 ]);
 </code-example>
 
-Congratulations! You are running a hybrid application! The existing AngularJS code works as before
+The existing AngularJS code works as before
```

Comment:
😞

Reply:
I know, I know. **sympathetic smile** 

It was such a happy moment calling for celebration. (Really, we're just fine-tuning some of the tone in the docs).